tests/test_qubit_device.py

```diff
@@ -326,31 +326,6 @@ def test_parameters_accessed_outside_execution_context(self, mock_qubit_device):
 class TestExtractStatistics:
     """Test the statistics method"""
 
-    def test_observables_deprecated(self, mock_qubit_device_extract_stats, monkeypatch):
```

Comment:
Is this related to the return types?

Reply:
Yes we never added observables to statistics, so no deprecation here.